Bottom: 5c63c5eb42120bdd707682e9fb308975adb078d9
Top:    4b83dd6516615a5e9c8a52c9998ee05adec841ba
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-04 18:28:18 -0700

Refresh of allow-arbitrary-python-streams

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 823733dd886..677c0738b8a 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -417,7 +417,7 @@ public:
 
   void Reset(PyRefType type, PyObject *py_obj) override;
 
-  llvm::StringRef GetString() const; // safe, null on error
+  llvm::StringRef GetString() const; // safe, empty string on error
 
   Expected<llvm::StringRef> AsUTF8() const;
